src/cat-data-service/src/service/api/health/mod.rs

```diff
@@ -16,12 +15,6 @@ impl HealthApi {
     /// This endpoint is used to determine if the service has started properly
     /// and is able to serve requests.
     ///
-    /// ## Responses
```

Comment:
I would rather we not remove the documentation about responses.
This is because the response themselves doesn't specifically tell you what you should do when you get the response.
This documentation does that.  The response documentation just shows you what that specific response looks like.
Otherwise reading the spec one has to guess what they should or shouldn't do when a `503` is returned.

Reply:
I see your point, will try to play with poem maybe it is possible to provide it directly into the response type. Or I will return back this description